core/src/main/java/com/scalar/db/common/error/CoreError.java

```diff
@@ -760,6 +760,10 @@ public enum CoreError implements ScalarDbError {
       "The underlying-storage data type %s is not supported as the ScalarDB %s data type: %s",
       "",
       ""),
+  DATA_LOADER_MISSING_NAMESPACE_OR_TABLE(
+      Category.USER_ERROR, "0165", "Missing namespace or table: %s, %s", "", ""),
+  DATA_LOADER_TABLE_METADATA_RETRIEVAL_FAILED(
+      Category.USER_ERROR, "0166", "Failed to retrieve table metadata. Details: %s", "", ""),
```

Comment:
When updating or creating error messages. Please explicitly ask @josh-wong to review them to check the English phrasing.
@josh-wong Please check the error message when you get the chance.